tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=linting,docs,py{37,38,39,310,311,py3},py{38}-pytest{main}
+envlist=docs,py{38,39,310,311,py3},py{38}-pytest{main}
```

Comment:
i presume linting was removed for pre-commit - it might be nice to have a changelog entry

Reply:
Yes it was removed in d128bc5872e2c2ec16da5abdf2c0163267985b4f.